adapters/InMobi/InMobiAdapter/GADMAdapterInMobiRewardedAd.h

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC.
+// Copyright 2022 Google LLC
```

Comment:
Revert back to `2022`, the same apples to all the other places.

Reply:
done